### PR TITLE
kamel 1.0.1

### PR DESCRIPTION
**Pull request by ipolyzos**
Upgrade kamel formula to v1.0.0 as releases of today

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Below you can find the logs for _build from source_  and _audit_ runs.

**brew --build-from-source**
```
Updating Homebrew...
==> Cloning https://github.com/apache/camel-k.git
Updating /Users/ipolyzos/Library/Caches/Homebrew/kamel--git
==> Checking out tag 1.0.1
HEAD is now at 21b88a3 Release 1.0.1
HEAD is now at 21b88a3 Release 1.0.1
Warning: A newer Command Line Tools release is available.
Update them from Software Update in System Preferences or run:
  softwareupdate --all --install --force

If that doesn't show you an update run:
  sudo rm -rf /Library/Developer/CommandLineTools
  sudo xcode-select --install

Alternatively, manually download them from:
  https://developer.apple.com/download/more/.

==> make
==> Caveats
Bash completion has been installed to:
  /usr/local/etc/bash_completion.d

zsh completions have been installed to:
  /usr/local/share/zsh/site-functions
==> Summary
🍺  /usr/local/Cellar/kamel/1.0.1: 8 files, 53.5MB, built in 1 minute 3 seconds
```

**brew test**
```
Testing kamel
==> /usr/local/Cellar/kamel/1.0.1/bin/kamel 2>&1
==> echo $(/usr/local/Cellar/kamel/1.0.1/bin/kamel help 2>&1)
==> echo $(/usr/local/Cellar/kamel/1.0.1/bin/kamel get 2>&1)
==> echo $(/usr/local/Cellar/kamel/1.0.1/bin/kamel version 2>&1)
==> echo $(/usr/local/Cellar/kamel/1.0.1/bin/kamel run 2>&1)
==> echo $(/usr/local/Cellar/kamel/1.0.1/bin/kamel run None.java 2>&1)
==> echo $(/usr/local/Cellar/kamel/1.0.1/bin/kamel reset 2>&1)
==> echo $(/usr/local/Cellar/kamel/1.0.1/bin/kamel rebuild 2>&1)
==> echo $(/usr/local/Cellar/kamel/1.0.1/bin/kamel reset 2>&1)
```

### CODE REVIEW

File: Formula/kamel.rb [hunk 1]
```diff
@@ -3,17 +3,10 @@ class Kamel < Formula
...
+    :revision => "21b88a3bfc07ae9fd9ff5f129ce3a22f1dfc5318"
   head "https://github.com/apache/camel-k.git"
 
-  bottle do
-    cellar :any_skip_relocation
-    sha256 "7ec8586715a660ab0b640c842e582db9cd7c5a533c0cfe6d8adf6b794018cbaa" => :catalina
-    sha256 "56c045f37e84b5bbad71e9eb46fedd2e7e4db914a6212d46276b06dcb0cb5403" => :mojave
-    sha256 "ba15c2fb6f75861c545c2a4c013b855b49a3238f352afaa5c71ff3437f297405" => :high_sierra
-  end
-
```
Comment (SMillerDev) [inline Formula/kamel.rb:16]: Please don't edit the bottle block

File: Formula/kamel.rb [hunk 2]
```diff
@@ -41,7 +34,7 @@ def install
     assert_match "Error: cannot get command client: could not locate a kubeconfig", get_output
 
     version_output = shell_output("echo $(#{bin}/kamel version 2>&1)")
-    assert_match "1.0.0", version_output
+    assert_match "1.0.1", version_output
```
Comment (SMillerDev) [inline Formula/kamel.rb:37]:
```suggestion
    assert_match version.to_s, version_output
```

File: Formula/kamel.rb (hunk 1 above)
Comment (ipolyzos) [reply Formula/kamel.rb]: > Please don't edit the bottle block

I reverted the _bottle_ block.

File: Formula/kamel.rb (hunk 2 above)
Comment (ipolyzos) [reply Formula/kamel.rb]: thanks for the suggestion, I have made use of `version.to_s` now

**Review by chenrui333 — APPROVED** [review]

**Pull request closed without merging**